code/_onclick/item_attack.dm

```diff
@@ -67,9 +67,6 @@
 	if(.)
 		return TRUE
 
-	if(user.a_intent != INTENT_HARM)
-		return
```

Comment:
You have not thought this through at all. Why are you changing this at the object level.